google/cloud/bigtable/internal/query_plan.h

```diff
@@ -37,8 +37,10 @@ class QueryPlan : public std::enable_shared_from_this<QueryPlan> {
   static std::shared_ptr<QueryPlan> Create(
       CompletionQueue cq, google::bigtable::v2::PrepareQueryResponse response,
       RefreshFn fn) {
-    return std::shared_ptr<QueryPlan>(
+    auto plan = std::shared_ptr<QueryPlan>(
```

Comment:
Let's go ahead and move the implementation of this function into the .cc file along with the other member functions.